spec/lib/diaspora/mentionable_spec.rb

```diff
@@ -182,7 +184,7 @@
         user_a.aspects.where(name: "generic").first.contacts.map(&:person_id)
       )
 
-      expect(txt).to include("@[user C](#{local_or_remote_person_path(user_c.person)}")
+      expect(txt).to include("[user C](#{local_or_remote_person_path(user_c.person)}")
```

Comment:
The advantage with that is we visually have a difference between the users who have been mentioned (so notified) and the ones you have a link to but can't access the post.